source/CardGames.py

```diff
@@ -345,7 +345,42 @@ def winnnerCheck(self, house, player):
     
     else:
       return 1 if houseHand <= 21 else 2
-
+# add the inital cards (4 cards - 1 face down card) to card count list at the beginning of every turn
+# input: the person hand list, totalCard list, and deck.size
+# output: 2 elements list of running count and true count value
+def cardCountIni(playerHand, totalCard, deckSize):
+  for card in playerHand:
+    count = 0
+    if card.value in [2,3,4,5,6]:
+      count = 1
+    elif card.value in [1,10,11]:
+      count = -1
+    totalCard.append(count)
+  runningCount = sum(totalCard)
+  trueCount = runningCount / (deckSize/52)
+  return [runningCount, trueCount]
+
+# add new card to card count list
+# input: the person hand list, totalCard list, and deck.size
+# output: 2 elements list of running count and true count value
+def cardCount(player, totalCard, deckSize):
+  count = 0
+  card = player.hand[-1]
+  if card in [2,3,4,5,6]:
```

Comment:
I was getting errors in the totalCard list where card values were being counted as 0 after my first two cards. I found the bug: The card object should specify the attribute value i.e. card.value and is missing in both if and elif statements in the cardCount method.

Reply:
Good catch